testing/bsg_cache/regression_non_blocking/sv.include

```diff
@@ -55,4 +55,11 @@ $BASEJUMP_STL_DIR/bsg_cache/bsg_cache_non_blocking_dma.v
 $BASEJUMP_STL_DIR/bsg_cache/bsg_cache_non_blocking.v
```

Comment:
Can you make this a makefile fragment instead of a VCS filelist? This way, if someone else edits and re-runs your tests, they won't have to run `make clean` between each iteration

Reply:
the problem with makefile fragment is that if some file is missing, it doesn't say which one is missing. I prefer having it this way, because I can continue to run tests even after having some changes.

Reply:
I'm afraid I don't understand what you mean - what doesn't say which one is missing? `make`? `vcs`?

Reply:
`make`

Reply:
If you include the files in the target dependencies though, it will tell you it can't find that file.

Reply:
the message from `make` is not very helpful. It will simple say

make: *** No rule to make target `4x4/simv'.  Stop.

I don't think automatically rebuilding is a critical feature.

Reply:
Is it a pattern rule?

Reply:
no it's not. I could delete one of the files expected in the filelist to create this message.